presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergMetadata.java

```diff
@@ -564,4 +581,90 @@ public void rollback()
                         table.getPredicate().intersect(newDomain)),
                 constraint.getSummary()));
     }
+
+    @Override
+    public void createView(ConnectorSession session, SchemaTableName viewName, ConnectorViewDefinition definition, boolean replace)
+    {
+        HiveIdentity identity = new HiveIdentity(session);
+        Map<String, String> properties = ImmutableMap.<String, String>builder()
+                .put(TABLE_COMMENT, "Presto View")
+                .put(PRESTO_VIEW_FLAG, "true")
+                .put(PRESTO_VERSION_NAME, "testversion")
+                .put(PRESTO_QUERY_ID_NAME, session.getQueryId())
+                .build();
+
+        Column dummyColumn = new Column("dummy", HIVE_STRING, Optional.empty());
+
+        Table.Builder tableBuilder = Table.builder()
+                .setDatabaseName(viewName.getSchemaName())
+                .setTableName(viewName.getTableName())
+                .setOwner(session.getUser())
+                .setTableType(org.apache.hadoop.hive.metastore.TableType.VIRTUAL_VIEW.name())
+                .setDataColumns(ImmutableList.of(dummyColumn))
+                .setPartitionColumns(ImmutableList.of())
+                .setParameters(properties)
+                .setViewOriginalText(Optional.of(encodeViewData(definition)))
+                .setViewExpandedText(Optional.of("/* Presto View */"));
+
+        tableBuilder.getStorageBuilder()
```

Comment:
We can simplify this using `withStorage`:
```java
Table table = Table.builder()
        .setDatabaseName(...)
        ...
        .withStorage(storage -> storage
                .setStorageFormat(VIEW_STORAGE_FORMAT)
                .setLocation(""))
        .build()
```
We should also try to create shared utility methods to create the `Table` object and deserialize it to `ConnectorViewDefinition` so that they are guaranteed to be the same between Hive and Iceberg, since the same metastore can be used for both (and thus the view formats should be the same).